client/blocks/signup-form/index.jsx

```diff
@@ -622,7 +623,28 @@ class SignupForm extends Component {
 												onClick={ ( event ) => this.handleLoginClick( event, fieldValue ) }
 											/>
 										),
-										pwdResetLink: <a href={ lostPassword( this.props.locale ) } />,
+										pwdResetLink: isReactLostPasswordScreenEnabled() ? (
+											<a
+												href="/"
+												onClick={ ( event ) => {
+													event.preventDefault();
+													recordTracksEvent( 'calypso_login_reset_password_link_click' );
```

Comment:
I'm seeing this tracks being used by other login pages. Should we use `calypso_signup_reset_password_link_click` instead?

Reply:
@ilyasfoo Good catch! Updated in [Use different track name](https://github.com/Automattic/wp-calypso/pull/95002/commits/3c884e3802a1f07be852bb73b989d77ca1bfed22)